contractNameAndAddress.ts

```diff
@@ -0,0 +1,17 @@
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+
+const func: GetContractNames = async function (hre: HardhatRuntimeEnvironment) {
```

Comment:
@t0rbik

Reply:
> @t0rbik

Script was introduced in [this commit](https://github.com/Velocimeter/instruments/pull/5/commits/8a2574389af638ffd8e8210a4d4fba831785e5cd).
It can be safely deleted as we don't use it, the type is obv wrong and doesn't exists in scope.

Reply:
I'm going to delete it.